hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -222,6 +223,7 @@ public class HoodieTableMetadataUtil {
   private static final int DECIMAL_MAX_PRECISION = 30;
   private static final int DECIMAL_MAX_SCALE = 15;
 
+  private static final DeleteContext DELETE_CONTEXT = new DeleteContext(CollectionUtils.emptyProps(), HoodieMetadataRecord.getClassSchema()).withReaderSchema(HoodieMetadataRecord.getClassSchema());
```

Comment:
guess this is just a dummy context, the metadata payload has its own isDelete logic.

Reply:
Yes, that's correct